### DISCUSSION TITLE
I am not able to create a Gatsby site. Keep getting the same error: Command failed with Unknown system error -86: git clone

### DISCUSSION THREAD

**Discussion by hansvmourik** [gatsbyjs/gatsby#36531] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hello,

I've been trying to create my first Gatsby site for over a week now. When I - for example - use the command gatsby new, and follow all the steps, everything seems to work until I answer the final question:  Create a new Gatsby site in the folder 'surfawhile'  with 'yes'.

Whatever I try, I receive the same error:

Command failed with Unknown system error -86: git clone https://github.com/gatsbyjs/gatsby-starter-minimal.git
/Users/hans/gatsby/surfawhile --recursive --depth=1 --quiet
spawn Unknown system error -86

What is going wrong here? 

I hope that somebody can point me to the right direction. 

With kind regards, 

Hans 

### Reproduction Link

https://codesandbox.io/s/github/gatsbyjs/gatsby-starter-minimal

### Steps to Reproduce

1. When I - for example - use the command gatsby new, 
2. Answer all the questions
3. UntilI answer the final question:  Create a new Gatsby site in the folder 'surfawhile'  with 'yes'.

### Expected Result

The expected result is the installation of the site in the right folder, which makes it possible to start developing in Gatsby. 

### Actual Result

Instead I receive the following error:

Command failed with Unknown system error -86: git clone https://github.com/gatsbyjs/gatsby-starter-minimal.git
/Users/hans/surfawhile --recursive --depth=1 --quiet
spawn Unknown system error -86

### Environment

```shell
System:
    OS: macOS 12.5.1
    CPU: (8) arm64 Apple M1
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.17.0 - /usr/local/bin/node
    npm: 8.15.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 104.0.5112.101
    Safari: 15.6.1
```

**Comment by LekoArts** [comment]
Hi!

When googling your errors I found these which seem relevant:

- https://stackoverflow.com/questions/70987542/webpack-cli-error-spawn-unknown-system-error-86-symfony
- https://github.com/imagemin/imagemin-mozjpeg/issues/61
- https://github.com/cypress-io/cypress/issues/20052

I'm also on an M1 Mac and the command works fine for me so somewhere your local setup needs an update.

Try (re-)installing the xcode tools (https://www.gatsbyjs.com/docs/tutorial/part-0/#install-xcode-command-line-tools) and your `git`.
